src/main/java/com/coveo/nashorn_modules/Require.java

```diff
@@ -21,4 +27,17 @@ public static Module enable(NashornScriptEngine engine, Folder folder) throws Sc
 
     return created;
   }
+
+  public static void registerHandler(FileHandler fileHandler) {
```

Comment:
I think it'd be better to pass the list of handlers to the `enable` call instead of relying on a global list that is shared everywhere in the process.